homeassistant/components/switchbee/switch.py

```diff
@@ -75,11 +74,32 @@ def __init__(self, hass, device: SwitchBeeBaseDevice, coordinator):
             ),
         )
 
+    @property
+    def available(self) -> bool:
+        """Return True if entity is available."""
+        return self._is_online
```

Comment:
This should probably take the coordinator attribute `last_update_success` in to consideration. See the `CoordinatorEntity` class for the default.